package.json

```diff
@@ -25,39 +25,39 @@
   },
   "dependencies": {
     "@angular/cdk": "^12.1.0",
-    "@ant-design/icons-angular": "^12.0.3",
+    "@ant-design/icons-angular": "^13.0.0-beta.0",
     "date-fns": "^2.16.1",
     "ngx-hover-preload": "0.0.3",
     "resize-observer-polyfill": "^1.5.1"
```

Comment:
Consider use https://github.com/juggle/resize-observer instead cuz resize-observer-polyfill seems deserted. Or drop this polyfill cuz Angular v13 dropped support for IE11.

Reply:
I think we can remove it because it is only used for IE, right?